desktop.js

```diff
@@ -1,15 +1,15 @@
 function fillGrid(n) {
-    var result = [];
-    var innerArray = [];
-    var count = 1
-    for (i = 0; i < n; i++){
-        for (var j = 0; j < n; j++){
-            innerArray.push(count)
-            count++;
-        }
-         result.push(innerArray)
-    }
-    return result;
+  var result = [];
+  var innerArray = [];
+var count = 1
```

Comment:
Did you intend to delete the spaces here?

Reply:
Oh, oops! I updated and pushed up a fix.